lib/data/models/speaker_model.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:nowingoogle/data/models/user_model.dart';
+import 'package:nowingoogle/domain/entities/speaker.dart';
+import 'package:nowingoogle/domain/enums/user_roles.dart';
+import 'package:nowingoogle/presentation/pages/constants/color_scheme.dart';
+
+class SpeakerModel extends UserModel {
```

Comment:
Will there be any user identifier in this model(uid) so that just with that we can show the user page in future, or can we reuse the email for the same case?